src/app/Debug.tsx

```diff
@@ -33,9 +33,9 @@ export class Debug extends React.Component<RootState, State> {
   }
 
   updateLogs = async () => {
-    const reactNativeLogs = await Logger.getMonthLogs()
     this.setState({
-      reactNativeLogs: reactNativeLogs || 'Not Found',
+      reactNativeLogs:
+        "This feature has been removed. It didn't work with log files above a certain size.",
```

Comment:
I don't think anyone on the team is really using that debug screen to read logs. But let me know if you do.

Reply:
can you just remove the feature? this seems like it leaves code hanging around that doesn't actually do anything anymore

Reply:
Yes, I first wanted to do this separately, but the whole log view is now removed.